ui/app.yaml

```diff
@@ -1,6 +1,4 @@
-runtime: python27
-api_version: 1
```

Comment:
what do these changes do?

Reply:
I could not find documentation for `api_version`, so I'm not sure about that one.. I believe `threadsafe` tells GAE that, even though I'm using Python, you can send two requests simultaneously because I've tried to keep my code thread-safe. This has never been necessary for Node to the best of my knowledge.

Reply:
ok